### DISCUSSION TITLE
1

### DISCUSSION THREAD

**Discussion by chosen179** [getlantern/lantern-forum-cn#405] [Q&A]

**Comment by MaxMKMax** [comment]
@chosen179 
请您提供我您的账号邮箱和购买单据. 我来帮您提交!

**Reply by MaxMKMax** [reply]
@chosen179 谢谢您支持蓝灯! 希望以后还能看见您. 

对了, 关于无法使用的问题可能您不使用最新版本. 请您下载最新版本适用看看. 那可能就不需要退款了. 希望您给我们一次机会. 谢谢! 

这两个链接都可以下载. 这个版本 7.8.4 应该修复了闪退和别的问题了. 

https://github.com/getlantern/lantern/blob/devel/README.zh.md

https://github.com/getlantern/lantern
